resources/assets/less/bem/mp-history-game.less

```diff
@@ -115,17 +137,23 @@
 
   &__player-score {
     width: 100%;
+    height: 125px;
 
     background-color: #ffffff;
 
     margin-bottom: 5px;
 
     .default-border-radius();
     .default-box-shadow();
+
+    @media @desktop{
```

Comment:
not sure if useful (2)

Coupled with removing `height: 100%;` in `mp-history-player-score__info-box`.

Reply:
oh, just if I removed `height: 100%;` in `mp-history-player-score__info-box` it should fix it, so no more `height: 125px;` in `mp-history-player-score`?